contracts/bls/BLSSignatureAggregator.sol

```diff
@@ -124,7 +122,7 @@ contract BLSSignatureAggregator is IAggregator {
     external view returns (bytes memory sigForUserOp) {
         uint256[2] memory signature = abi.decode(userOp.signature, (uint256[2]));
         uint256[4] memory pubkey = getUserOpPublicKey(userOp);
-        uint256[2] memory message = userOpToMessage(userOp);
+        uint256[2] memory message = _userOpToMessage(userOp, _getPublicKeyHash(pubkey));
```

Comment:
Did you replace `userOpToMessage` here with the code that does the same to avoid repeated call to the  `getUserOpPublicKey`? I think this breaks the idea of having the same code be reusable.
If this is not too late to change, maybe it could make sense to return `pubkey`, `hashPublicKey` and `message` from `userOpToMessage()` helper?

Reply:
Yes, its possible, but I dislike returning unused data. returning 3 items means that each call-site receives return values it never uses